rules/branch-naming/rule.md

```diff
@@ -0,0 +1,78 @@
+---
+type: rule
+tips: ""
+title: Do your branches have a naming convention?
+seoDescription: Using effective branch naming conventions with PBI numbers, descriptive names, and structured formats to enhance collaboration and traceability.
+uri: branch-naming
+authors:
+  - title: Christian Morford-Waite
+    url: https://ssw.com.au/people/christian-morford-waite
+related:
+  - do-you-know-when-to-branch-in-git
+  - clear-meaningful-names
+created: 2025-01-02T15:40:00.000Z
+guid: a97fa64d-7684-4220-8042-0b3be889aa9d
+---
+
+Have you ever looked at the list of branches in your repository and not know what they were for?
+
+Consistent naming conventions in source control systems help improve collaboration, clarity, and organization across teams. Using meaningful branch names with PBI numbers enhances traceability and reduces [technical debt](/technical-debt).
+
+<!--endintro-->
+
+Always remember to:
+
+- **Include a PBI number** - This makes it easy to track the purpose of the branch, reduces confusion, and provides context
+- **Use lowercase kebab style naming** - Keeps the name easy to read and compatible with a wide range of systems
+- **Be descriptive** - Clearly indicate the purpose or content of the branch
+- **Keep it short** - Use concise yet meaningful names
+- **Extra: Use categories for improved organization** - If your repository has many branches or you want to keep them organized, adding a prefix such as "feature", "bugfix", or "hotfix" is beneficial
+
+::: info
+**Note:** Release branches should always include a prefix (e.g., release/) to distinguish them from other branch types and keep them organized.
+:::
+
+::: greybox
+{{ CATEGORY }}/{{ PBI NUMBER }}-{{ DESCRIPTION }}   
+:::
+::: good
+Figure: Good example - Recommended naming format
+:::
+
+---
+
+::: greybox
```

Comment:
Move the bad example to be above the good example (I'm pretty sure we do bad -> good for rules).